src/assisted_test_infra/test_infra/controllers/ipxe_controller/server/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3
```

Comment:
@osherdp Not sure if this is the right path for this file, WDYT?

Reply:
might cause problems with rate-limiters of docker-hub
meaning we should strive to using quay.io, otherwise CI will get blocked because of its high usage of docker-hub

Reply:
I couldn't find `python:3.9.12-alpine3.15` in a quay repository. 

Should I tag and push the docker-hub image into my own repo and use that? The Dockerfile would look like this:
```
FROM quay.io/cchun/python:3.9.12-alpine3.15
```

Is there a better or more preferred way to do this?

Reply:
maybe there is another way to do that, but I think that will suffice 
go ahead and save it in your repo (just make sure you're toggling it public)

Reply:
Added it to my repo and made the change to the Dockerfile. Thanks!